docs/basics/index.md

```diff
@@ -238,7 +238,7 @@ model for a total of 64 samples)
 First prepare evaluation data
 
 ```bash
-python -m nemo_skills.dataset.prepare aime24 aime25
+ns prepare_data aime24 aime25
```

Comment:
Hmm does the script work for local data preparation too? If not, lets show both examples for local prep and cluster prep

Reply:
Scratch that, it works for local too.